llvm/test/MachineVerifier/copy-scalable.mir

```diff
@@ -38,15 +32,10 @@ body:             |
 ...
 
 ---
-name:            test_copy_virtual_to_physical
-legalized:       true
-regBankSelected: false
-selected:        false
-tracksRegLiveness: true
```

Comment:
tracksRegLiveness is annoyingly false by default (I really don't understand why this is an option). For now I'd leave it for any test that has a physical register reference in it